basics/answers.md

```diff
@@ -296,6 +307,29 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(converted_numbers)
     ```
 
+5. ``` python
+   rows=int(input("Type the number of rows needed:")) 
+   for row in range(1,rows+1):
+       for column in range(1,row+1): 
+           print(column,end=" ") 
+       print()#creates a new line for the next row
```

Comment:
There's an explanation below where you basically explain what `rows` and `row` mean. I think it would be better to use better variable names instead. (I also added some whitespace.)

```suggestion
   row_count = int(input("Type the number of rows needed:")) 
   for column_count in range(1, row_count+1):
       for number in range(1, column_count+1): 
           print(number, end=" ") 
       print()  # creates a new line for the next row
```

You will also need to change the other answer and update the explanations.